gems/smithy/lib/smithy/rivets/rubocop.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Rivets
+    # Runs RuboCop autocorrect-all on the generated projections.
+    class Rubocop < Rivet
+      TEST_NAMESPACES = [
```

Comment:
Since polishes and welds are now the same concept, we can't skip them selectively in tests, because we will still want to load them. We can instead use for?(service) (instead of model) as a way to select (can also look at the plan if needed). We can make all of the tests use smithy.ruby.tests (requires manual sync) or perhaps some upstream fixes to make the tests follow standard naming.